Dockerfile

```diff
@@ -1,64 +1,56 @@
-FROM phusion/passenger-full:1.0.9
-LABEL maintainer="kgarza@datacite.org"
+FROM phusion/baseimage:0.11
+LABEL maintainer="mfenner@datacite.org"
+
```

Comment:
"`Live-reload`" doesn't seem to work anymore. When running docker-compose, If I make changes in the code; they do not seem reflected in the running app. One needs to rebuild the whole container.